label.py

```diff
@@ -116,6 +119,27 @@ def create_change_shape(drawing_area, start_x, start_y, end_x, end_y):
     cv2.arrowedLine(drawing_area, (end_x - 2, end_y - 2), (end_x - 12, end_y - 12), (255, 0, 0), 2, tipLength=0.3)
 
 
+def draw_bounding_boxes_init(param):
+    [main_drawing_area, secondary_drawing_area, boxes] = param
+    # GREEN --> 'n'     YELLOW --> 'd'
+    color = {'d': (0, 255, 255), 'n': (0, 255, 0)}
```

Comment:
Define the colors at the beginning as constants, so you can change them for all places in one go.
COLOR_N = (0, 255, 0)  # GREEN
COLOR_D = (0, 255, 255)  # YELLOW